src/configurations/destinations/userpilot/db-config.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "USERPILOT",
+  "displayName": "Userpilot",
+  "config": {
+    "transformAtV1": "processor",
+    "saveDestinationResponse": true,
+    "includeKeys": [
+      "oneTrustCookieCategories",
+      "token",
+      "endpoint",
+      "apiEndpoint",
+      "apiKey",
+      "connectionMode",
+      "ketchConsentPurposes",
+      "consentManagement"
+    ],
+    "excludeKeys": [],
+    "supportedSourceTypes": ["web", "cloud"],
+    "supportedConnectionModes": {
+      "web": ["cloud", "device"],
+      "cloud": ["cloud"]
+    },
+    "supportedMessageTypes": {
+      "device": {
+        "web": ["identify", "track", "page", "group"]
+      },
+      "cloud": ["identify", "track", "group"]
+    },
+    "destConfig": {
+      "defaultConfig": ["token", "apiKey"],
+      "web": [
+        "endpoint",
```

Comment:
better to call sdkEndpoint or appEndpoint

Reply:
updated